deploy/load-test/stack/web-interface/app.yml

```diff
@@ -29,7 +29,7 @@ services:
                     - "node.labels.long-running==1"
     web-interface-publish-running-games-count-to-nchan:
```

Comment:
This service definition needs to move to `deploy/load-test/stack/web-interface/...yml`.